CMakeLists.txt

```diff
@@ -255,7 +255,7 @@ endif()
 option(REALM_ENABLE_MEMDEBUG "Add additional memory checks" OFF)
 option(REALM_VALGRIND "Tell the test suite we are running with valgrind" OFF)
 option(REALM_METRICS "Enable various metric tracking" ON)
-option(REALM_ENABLE_SYNC_MULTIPLEXING "Enables sync session multi-plexing by default")
+option(REALM_DISABLE_SYNC_MULTIPLEXING "Disables sync session multi-plexing by default")
```

Comment:
it's a bit weird for an option to be enable/disable in the name. Implicitly this is set to OFF by 'option'. Why not to just make it as 'REALM_METRICS' above and others? Where the default and the value for an option defines if it's active or inactive?

Reply:
Sure. We aren't super consistent around this - we have `REALM_ENABLE_GEOSPATIAL` and `REALM_METRICS`. But I've made this change for you.